src/MeshlessFV/MeshlessFVSimulation.cpp

```diff
@@ -462,19 +502,41 @@ void MeshlessFVSimulation<ndim>::PostInitialConditionsSetup(void)
     mfv->InitialSmoothingLengthGuess();
     mfvneib->BuildTree(rebuild_tree, 0, ntreebuildstep, ntreestockstep, mfv->Ntot,
                        mfv->Nhydromax, timestep, partdata, mfv);
+#ifdef MPI_PARALLEL
+  mfvneib->InitialiseCellWorkCounters();
+#endif
```

Comment:
Do we need to set the active particle counters and update_density flags here before we update all properties, like we do below. If not, why not?